util.js

```diff
@@ -717,3 +717,10 @@ exports.callbackify = callbackify;
 const textEncoding = require('text-encoding-polyfill');
```

Comment:
Here is the NPM link to [`text-encoding-polyfill`](https://www.npmjs.com/package/text-encoding-polyfill). There is some healthy traffic downloading this dependency. From a quick glance, the code looks safe